test/uk/gov/hmrc/customs/financials/emailthrottler/services/EmailJobHandlerSpec.scala

```diff
@@ -37,11 +37,19 @@ class EmailJobHandlerSpec extends SpecBase {
 
```

Comment:
Could you please address below for this file?
1. 102 and 106 are long lines. PLease split over multiple lines
2. Remove extra line break at line no 129